WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -328,11 +328,32 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleGetWindowSize:(FBRouteRequest *)request
 {
-  CGRect frame = request.session.application.wdFrame;
-  return FBResponseWithStatus(FBCommandStatusNoError, @{
-    @"width": @(CGRectGetWidth(frame)),
-    @"height": @(CGRectGetHeight(frame)),
-  });
+    CGRect frame = request.session.application.wdFrame;
+    NSNumber *width = @(CGRectGetWidth(frame));
+    NSNumber *height = @(CGRectGetHeight(frame));
+    NSNumber *orientationValue = @0;
+
+    UIInterfaceOrientation orientation = request.session.application.interfaceOrientation;
+    if(orientation == UIInterfaceOrientationPortrait) { //Portrait orientation
```

Comment:
1) Kill comments as they are redundant. 
2) Use switch instead, you can share a lot of code here.
3) Do you care about returning orientation?